corehq/toggles.py

```diff
@@ -507,7 +507,7 @@ def _ensure_valid_randomness(randomness):
     namespaces=[NAMESPACE_DOMAIN]
 )
 
-IS_DEVELOPER = StaticToggle(
+IS_CONTRACTOR = StaticToggle(
     'is_developer',
     'Is developer',
```

Comment:
This is what will show up on the feature flag page, could probably stand to change that too, or possibly to "Is developer/contractor" or something if you want to keep it sort-of in the same place.

Reply:
agreed, updated